### PR TITLE
comment listener/readability prototype

### PR DESCRIPTION
**Pull request by alexbyrnes**
Test of scale etc for doing readability scores. Integration points are rabbitmq (publisher is currently Pillar) and scores go to stdout.  Stdout would need to be replaced.

This also gives us another couple of data points for statsd.

### CODE REVIEW

File: commentListener.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    rmq_exchange = os.environ['AMQP_EXCHANGE']
+    statsd_host = os.environ['STATSD_HOST']
+    statsd_port = int(os.environ['STATSD_PORT'])
+
+    # For mongo integration
+    #mongo_db = os.environ['DBNAME']
+    #mongo_host = os.environ['MONGODB_HOST']
+except Exception as err:
+    print("ERROR: Environment variable not found. " + err, file=sys.stderr)
+    exit()
```
Comment (crccheck) [inline commentListener.py]: you should send a non-zero exit, like `exit(1)`, that way if you include this as part of some other tooling, they can infer something wrong happened by the exit code.

Comment (alexbyrnes) [inline commentListener.py]: Thanks, Chris. agreed.

**Comment by gabelula** [comment]
Great. Can we add some tests to this?

**Comment by alexbyrnes** [comment]
I'm going to close this in favor of whatever happens with [coral-stats branch](https://github.com/coralproject/atoll/compare/coral-stats). Will add tests there :)

**Pull request closed without merging**